src/test/java/com/easypost/ShipmentTest.java

```diff
@@ -344,14 +343,16 @@ public void testCreateTaxIdentifiers() throws EasyPostException {
      * @throws EasyPostException when the request fails.
      */
     @Test
-    @Disabled
-    // TODO: test is for some reason failing to pull a proper recording when playing
-    // back. Only test doing this
     public void testCreateWithIds() throws EasyPostException {
         vcr.setUpTest("create_with_ids");
 
-        Address fromAddress = vcr.client.address.create(Fixtures.caAddress1());
-        Address toAddress = vcr.client.address.create(Fixtures.caAddress1());
+        Map<String, Object> address1Data = Fixtures.caAddress1();
+        Map<String, Object> address2Data = Fixtures.caAddress2();
+        // VCR will overwrite the first address recording if the parameters are the exact same,
+        // which will cause us to lose the response from the first address creation and cause the replay to fail.
+        address2Data.put("name", "EasyPost");
```

Comment:
Question: Why the name bit here? This should be as simple as using the 2nd address because they are totally different fixtures. We shouldn't need to mess with the data further?

Reply:
You're right. I forgot we had two different address fixtures at our disposal, but CoPilot didn't.